### ISSUE TITLE
Twitch LogIn doesn't work

### ISSUE THREAD

**Issue by martinmei71** [elementary/triage#167]
### What Happened?

When I try to log in into Twitch at any browser as Google Chrome, Firefox or Edge, I can't accept cookies or login with my account. But other pages as PrimeVideo or HBOMax works propely

### Steps to Reproduce

Try to LogIn to Twitch at Google Chrome or Firefox using elementaryOS 7 as SO

### Expected Behavior

Shows a message that the browser is not compatible, when you try to log in.
Doesn't do nothing when you try to accept webpage cookies

### OS Version

5.x (Hera)

### Software Version

Latest release (I have run all updates)

**Comment by wasantosfi** [comment]
Hi @martinmei71 how are you?

Did you ever tried seeing if the repo (in the sources lists under /etc/apt/ path) of Google Chrome or Mozilla Firefox are referencing to the latest version at all? Eg.: They could be referencing old versions of Ubuntu/Debian, maybe.
Did you install those browsers via Flathub?
Try to purge them, download them after from Flathub and try to login again.

Here I use Twitch almost daily, but I use Brave Browser (for privacy reasons).

Best Regards.
Walter Filho

**Comment by martinmei71** [comment]
I intalled via Flathub and i have tried to reinstall them also in other PC and the same. 🙁

________________________________
De: wasantosfi ***@***.***>
Enviado: sábado, 4 de marzo de 2023 14:22
Para: elementary/triage ***@***.***>
Cc: Martin Meizoso ***@***.***>; Mention ***@***.***>
Asunto: Re: [elementary/triage] Twitch LogIn doesn't work (Issue #167)


Hi @martinmei71<https://github.com/martinmei71> how are you?

Did you ever tried seeing if the repo (in the sources lists under /etc/apt/ path) of Google Chrome or Mozilla Firefox are referencing to the latest version at all? Eg.: They could be referencing old versions of Ubuntu/Debian, maybe.
Did you install those browsers via Flathub?
Try to purge them, download them after from Flathub and try to login again.

Here I use Twitch almost daily, but I use Brave Browser (for privacy reasons).

Best Regards.
Walter Filho

**Comment by wasantosfi** [comment]
Hummm, curious.
Did you tried to login via smartphone app just to see if logs in?
Is MFA (Authenticator required) set up?
What is the other PC's system?

Best Regards.
Walter Filho

**Comment by martinmei71** [comment]
Yes on my phone i can log in, at KDE neon i can log in, also with windows it only happens in elementaryOS

________________________________
De: wasantosfi ***@***.***>
Enviado: sábado, 4 de marzo de 2023 14:49
Para: elementary/triage ***@***.***>
Cc: Martin Meizoso ***@***.***>; Mention ***@***.***>
Asunto: Re: [elementary/triage] Twitch LogIn doesn't work (Issue #167)


Hummm, curious.
Did you tried to login via smartphone app just to see if logs in?
Is MFA (Authenticator required) set up?
What is the other PC's system?

Best Regards.
Walter Filho

**Comment by wasantosfi** [comment]
Hummm, understood.

Have you tried using the Anonymous Tab in Google Chrome in order to see if it works?

Maybe trying Epiphany (embedded browser) or Opera Browser (from flathub), or Brave Browser (https://brave.com/pt-br/linux/).

Best Regards.
Walter Filho

**Comment by martinmei71** [comment]
Yes i have tried that options but same failure
________________________________
De: wasantosfi ***@***.***>
Enviado: sábado, 4 de marzo de 2023 14:57
Para: elementary/triage ***@***.***>
Cc: Martin Meizoso ***@***.***>; Mention ***@***.***>
Asunto: Re: [elementary/triage] Twitch LogIn doesn't work (Issue #167)


Hummm, understood.

Have you tried using the Anonymous Tab in Google Chrome in order to see if it works?

Maybe trying Epiphany (embedded browser) or Opera Browser (from flathub), or Brave Browser (https://brave.com/pt-br/linux/).

Best Regards.
Walter Filho

**Comment by martinmei71** [comment]
so now i'm using KDE neon to have not that errors

________________________________
De: Martin Meizoso ***@***.***>
Enviado: sábado, 4 de marzo de 2023 15:04
Para: elementary/triage ***@***.***>
Asunto: RE: [elementary/triage] Twitch LogIn doesn't work (Issue #167)

Yes i have tried that options but same failure
________________________________
De: wasantosfi ***@***.***>
Enviado: sábado, 4 de marzo de 2023 14:57
Para: elementary/triage ***@***.***>
Cc: Martin Meizoso ***@***.***>; Mention ***@***.***>
Asunto: Re: [elementary/triage] Twitch LogIn doesn't work (Issue #167)


Hummm, understood.

Have you tried using the Anonymous Tab in Google Chrome in order to see if it works?

Maybe trying Epiphany (embedded browser) or Opera Browser (from flathub), or Brave Browser (https://brave.com/pt-br/linux/).

Best Regards.
Walter Filho

**Comment by wasantosfi** [comment]
Got, unfortunately I don't know what else to do (I'm not specialist on it neither Elementary staff).

**Comment by martinmei71** [comment]
Don't worry i send the error to inform about it. 🙂
I thank you very much for the help.
________________________________
De: wasantosfi ***@***.***>
Enviado: sábado, 4 de marzo de 2023 15:07
Para: elementary/triage ***@***.***>
Cc: Martin Meizoso ***@***.***>; Mention ***@***.***>
Asunto: Re: [elementary/triage] Twitch LogIn doesn't work (Issue #167)


Got, unfortunately I don't know what else to do (I'm not specialist on it neither Elementary staff).

**Comment by danirabbit** [comment]
Closing since this is either an issue in a third party browser or with Twitch.tv but also I can't reproduce this. Logging into twitch with Web works as expected here
